fe/fe-core/src/main/java/org/apache/doris/datasource/property/storage/HDFSProperties.java

```diff
@@ -131,20 +148,20 @@ public void toHadoopConfiguration(Configuration conf) {
     }
 
     public Configuration getHadoopConfiguration() {
-        Configuration conf = new Configuration(false);
+        Configuration conf = new Configuration(true);
         Map<String, String> allProps = loadConfigFromFile(getResourceConfigPropName());
         allProps.forEach(conf::set);
         if (MapUtils.isNotEmpty(finalHdfsConfig)) {
             finalHdfsConfig.forEach(conf::set);
         }
-        conf.set("hdfs.security.authentication", hdfsAuthenticationType);
```

Comment:
Here we need to determine the behavior of be. What parameter format does be take? Same as before?